configs/warnings.tslint.json

```diff
@@ -1,8 +1,10 @@
 {
   "rules": {
+    "await-promise": true,
     "no-any": {
       "severity": "warning",
       "options": true
-    }
+    },
+    "no-return-await": true
```

Comment:
Maybe it is me being picky, but I disagree with this rule in particular: https://palantir.github.io/tslint/rules/no-return-await/

> An async function always wraps the return value in a Promise. Using return await just adds extra time before the overreaching promise is resolved without changing the semantics.

I feel like the semantics are a bit changed in some corner cases (but that appear in our code):

```js
async function thrower() {
    throw new Error('throwed')
}

async function case1() {
    try {
        return await thrower()
    } catch (e) {
        console.log('1', e.message)
    }
}

async function case2() {
    try {
        return thrower()
    } catch (e) {
        console.log('2', e.message)
    }
}

case1()
case2()
```
Output:
```
1 throwed
(node:32467) UnhandledPromiseRejectionWarning: Error: throwed
    at thrower (/home/emaapur/projects/easy-cpp/prout.js:2:11)
    at case2 (/home/emaapur/projects/easy-cpp/prout.js:15:16)
    at Object.<anonymous> (/home/emaapur/projects/easy-cpp/prout.js:22:1)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:191:16)
(node:32467) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
(node:32467) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

Reply:
I would be good with it!